docs/concepts/what-is-an-event.md

```diff
@@ -86,131 +167,357 @@ It is possible for the following events to still occur:
 This event type indicates that the Activity has timed out according to the Temporal Server, due to the Activity having not completed within the timeout settings.
 
+| Field Name         | Type       | Definition |
+| ------------------ | ---------- | ---------- |
+| failure            | Failure    | definition |
+| scheduled_event_id | int64      | definition |
+| started_event_id   | int64      | definition |
+| retry_state        | RetryState | definition |
+
 ### ActivityTaskCancelRequested
 
 This event type indicates that a request to cancel the Activity has occurred.
 The SDK client will be able to confirm cancellation of an Activity during an Activity heartbeat.
 
+| Field Name                       | Type  | Definition |
+| -------------------------------- | ----- | ---------- |
+| scheduled_event_id               | int64 | definition |
+| workflow_task_completed_event_id | int64 | definition |
+
 ### ActivityTaskCanceled
 
 This event type indicates that the Activity has been canceled.
 
+| Field Name                       | Type     | Definition                                                                                      |
+| -------------------------------- | -------- | ----------------------------------------------------------------------------------------------- |
+| details                          | Payloads | Additional information reported by the Activity upon confirming cancellation.                   |
+| latest_cancel_requested_event_id | int64    | ID of the most recent 'ACTIVITY_TASK_CANCEL_REQUESTED' event which refers to the same Activity. |
+| scheduled_event_id               | int64    | the ID of the 'ACTIVITY_TASK_SCHEDULED' event this cancel confirmation corresponds to.          |
+| started_event_id                 | int64    | definition                                                                                      |
+| identity                         | string   | definition                                                                                      |
+
 ### TimerStarted
 
 This event type indicates a timer has started.
 
+| Field Name                       | Type     | Definition                                       |
+| -------------------------------- | -------- | ------------------------------------------------ |
+| timer_id                         | string   | The Worker or User ID assigned for the timer.    |
+| start_to_fire_timeout            | Duration | Amount of time to elapse before the timer fires. |
+| workflow_task_completed_event_id | int64    | definition                                       |
+
 ### TimerFired
 
 This event type indicates a timer has fired.
 
+| Field Name       | Type   | Definition |
+| ---------------- | ------ | ---------- |
+| timer_id         | string | definition |
+| started_event_id | int64  | definition |
+
 ### TimerCanceled
 
 This event type indicates a Timer has been canceled.
 
+| Field Name                       | Type       | Definition |
```

Comment:
`Field Name` should be `Field name` due to [sentence casing](https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings)

Reply:
Drop "name" altogether in favor of just "Field"